src/plugin.json

```diff
@@ -68,6 +68,11 @@
         "targets": ["grafana-metricsdrilldown-app/grafana-assistant-app/navigateToDrilldown/v0-alpha"],
         "title": "Navigate to metrics drilldown",
         "description": "Build a url path to the metrics drilldown"
+      },
+      {
+        "targets": ["grafana/datasources/config/actions", "grafana/datasources/config/status"],
+        "title": "Open in Metrics Drilldown",
+        "description": "Browse metrics in Grafana Metrics Drilldown"
```

Comment:
This is the main change, which needs to match the same targets, title and description as in the `datasourceConfigLinkConfigs` function so it to work properly